src/Compilers/Test/Core/Compilation/OperationTreeVerifier.cs

```diff
@@ -1941,7 +1941,7 @@ public override void VisitDiscardPattern(IDiscardPatternOperation operation)
 
         public override void VisitSwitchExpression(ISwitchExpressionOperation operation)
         {
-            LogString($"{nameof(ISwitchExpressionOperation)} ({operation.Arms.Length} arms)");
+            LogString($"{nameof(ISwitchExpressionOperation)} ({operation.Arms.Length} arms, IsExhaustive: {operation.IsExhaustive})");
```

Comment:
>operation.IsExhaustive [](http://example.com/codeflow?start=107&length=22)

We have another test visitor that should query this property too.

Reply:
which test do you mean?

Reply:
https://github.com/dotnet/roslyn/blob/main/src/Compilers/Test/Core/Compilation/TestOperationVisitor.cs#L1285-L1293 this method will want to be updated.